engines/grim/grim.cpp

```diff
@@ -261,47 +261,34 @@ LuaBase *GrimEngine::createLua() {
 
 GfxBase *GrimEngine::createRenderer(int screenW, int screenH) {
 	Common::String rendererConfig = ConfMan.get("renderer");
-	Graphics::RendererType desiredRendererType = Graphics::parseRendererTypeCode(rendererConfig);
-	Graphics::RendererType matchingRendererType = Graphics::getBestMatchingAvailableRendererType(desiredRendererType);
+	Graphics::RendererType desiredRendererType = Graphics::Renderer::parseTypeCode(rendererConfig);
+	unsigned int availableRendererTypes = Graphics::Renderer::getAvailableTypes();
```

Comment:
we may use uint

Reply:
I used uint32 to have something with fixed size.